simpleLooping/binaryListToString.py

```diff
@@ -15,3 +15,21 @@
 # Ex:
 #   binaryListToString([1, 0, 1, 1])       # return "1101"
 #   binaryListToString([0, 1, 1, 0, 0, 1]) # return "100110"
+from typing import List
+
+def binaryListToString(binaryList: List[int]) -> str:
+    lenBinaryList = len(binaryList)
+    
+    if lenBinaryList == 0:
+        return "-1"
```

Comment:
“-1” is not equal with -1. One is string and one is integer. Did you run the unit test? Did it return the correct result as you expected?